tests/ci/profile_handler.go

```diff
@@ -88,8 +89,16 @@ func PrepareRoute53() {}
 
 func GenerateClusterCreationArgsByProfile(profile *Profile) (clusterArgs *EXE.ClusterCreationArgs, manifestsDir string, err error) {
 	clusterArgs = &EXE.ClusterCreationArgs{
-		Token: os.Getenv(CON.TokenENVName),
+		Token: CON.TokenENVName,
 	}
+
+	if CON.ManifestsDirENV == "" {
+		// deafult profile dir selected
+		profile.ManifestsDIR = CON.ROSAClassic
```

Comment:
ManifestsDirENV is used to set the whole manifest dir path. Please don't set it to rosaclassic